src/beautifiers/eslint.coffee

```diff
@@ -23,12 +24,25 @@ module.exports = class ESLintFixer extends Beautifier
       allowUnsafeNewFunction ->
         importPath = Path.join(projectPath, 'node_modules', 'eslint')
         try
-          CLIEngine = require(importPath).CLIEngine
+          context.which('eslint').then((globalEslintPath) ->
+            if globalEslintPath != 'eslint'
+              importPath = Path.resolve(globalEslintPath, '../../lib/node_modules/eslint')
+            else
+              needInstallMsg = 'Error: it seems like you did not install eslint globally,'
+              needInstallMsg += 'try to run npm i -g eslint and restart the atom editor.'
+              context.deprecate needInstallMsg
```

Comment:
This message looks like it should be an error message to the user, not a deprecation notice.

Reply:
Understand.
Can I use `@verbose("error notification message")` here instead of `deprecation`?

Reply:
Yes, `@verbose(needInstallMsg)` should work. Let me know if you have any problems.